packages/driver/src/cy/actionability.coffee

```diff
@@ -232,17 +244,23 @@ verify = (cy, $el, options, callbacks) ->
 
     runAllChecks = ->
       if force isnt true
+        ## ensure its 'receivable'
+        if (options.ensure.receivability) then cy.ensureReceivability($el, _log)
```

Comment:
I know this naming convention is already in develop, but...I hate this name, I have to look directly at the method every time. And all is reasonably does is check that the el is not disabled. If we're adding options like `notReadonly` and `notCovered`, can't we just update this to say `notDisabled` and `ensureNotDisabled`?

Reply:
agreed, i should either adopt the "ensureWritable" name for readonly as you had in the original PR, or change this to `ensureNotDisabled`.

Which is  better?

Reply:
renamed to `ensureNotDisabled`